### PR TITLE
#164599870 Tests login, sign up, and get all messages codes

### PR DESCRIPTION
**Pull request by habinezadalvan**
what does this PR do?
Tests codes for login, signup, get messages endpoints
**Description of Task to be completed**
written tests for:
-login endpoint
-signup endpoint
-get all messages endpoint
**How should this be tested manually**
-clone repo
-install dependencies with npm install
-Run test with npm test
**relative pivotal tracker story**
#164599870
@CeciliaCaroline @Joe-Joseph @Prosper07  @nnyves

### CODE REVIEW

**Comment by Joe-Joseph** [comment]
Good job adding tests, take courage man, and finish the work

File: .env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+SECRET_KEY = "TOKENHELLOWORLD!!"
```
Comment (CeciliaCaroline) [inline .env:1]: You should not commit the .env file to github. Add it to the gitignore

File: coverage/server/controllers/index.html [hunk 2]
```diff
@@ -0,0 +1,110 @@
+<!doctype html>
```
Comment (CeciliaCaroline) [inline coverage/server/controllers/index.html:1]: You might have to add the coverage folder to the gitignore as well

File: server/controllers/account.js [hunk 3]
```diff
@@ -19,7 +19,10 @@ const account = {
...
     const { error } = signUpValidation.validateSignUp(req.body);
     if (error) {
-      res.status(400).send(error.details[0].message);
+      res.status(400).json({
+        status: 400,
+        error: error.details[0].message,
+      });
       return;
     }
     // the fellowing lines creates the id by adding 1 to whichever index number of requested user's data structure.
```
Comment (CeciliaCaroline) [inline server/controllers/account.js]: HAve more brief and precise comments

File: server/tests/account.js [hunk 4]
```diff
@@ -0,0 +1,104 @@
+/* eslint-disable import/no-extraneous-dependencies */
```
Comment (CeciliaCaroline) [inline server/tests/account.js:1]: Your tests files should be of the format `account.test.js`

**Review by CeciliaCaroline — COMMENTED** [review]
Please make suggested changes

File: .env (hunk 1 above)
Comment (habinezadalvan) [reply .env:1]: Yes, I am going to do it. I am going to do it, I thought I did.

**Comment by habinezadalvan** [comment]
Thank you for your insights

**Pull request merged**